.travis.yml

```diff
@@ -1,23 +1,9 @@
-language: android
-
-android:
-  components:
-    - build-tools-20.0.0
-    - android-17
-    - sys-img-armeabi-v7a-android-17
-  licenses:
-    - android-sdk-license-5be876d5
+language: java
 
 jdk:
   - oraclejdk7
   - oraclejdk8
 
-before_script:
-  - echo no | android create avd --force -n test -t android-17 --abi armeabi-v7a
-  - emulator -avd test -no-skin -no-audio -no-window &
-  - android-wait-for-emulator
-  - adb shell input keyevent 82 &
```

Comment:
Yeah... hadn't seen that script until now. Kind of impressed that you got it working.